ibis/backends/tests/test_numeric.py

```diff
@@ -1371,8 +1371,10 @@ def test_clip(backend, alltypes, df, ibis_func, pandas_func):
 )
```

Comment:
unrelated: I think we can remove the @pytest.mark.broken("druid")?

Reply:
Does it pass for you? We can only remove it if the test failure says `XPASS` in it, meaning it's actually **not** failing.

Reply:
Something is broken about my local testing environment and I haven't taken the time to fix it so I haven't actually run these. Just trying to rely on CI